src/components/Header.jsx

```diff
@@ -0,0 +1,17 @@
+import { Link } from 'react-router-dom';
+
+function Header() {
+    return (
+      <>
```

Comment:
Fragments don't tell anyone what this is supposed to represent, it would be skipped by a screen reader as unimportant. Consider what element would be better to use.